packages/@lwc/engine-server/src/__tests__/fixtures/lwc-dynamic/error-ssr.txt

```diff
@@ -0,0 +1,2 @@
+SSR compilation error:
+The lwc:dynamic directive is not supported for SSR. Use <lwc:component> instead.
```

Comment:
This is a good use for having a separate SSR-only fixture. Although I think we can probably safely drop `lwc:dynamic` from `engine-server` entirely.